tests/AppBundle/Controller/AdherentControllerTest.php

```diff
@@ -105,13 +105,14 @@ public function testEditProfileFillAdherentData()
         $errors = $crawler->filter('.form__errors > li');
 
         $this->assertResponseStatusCode(Response::HTTP_OK, $this->client->getResponse());
-        $this->assertSame(6, $errors->count());
-        $this->assertSame('Cette valeur ne doit pas être vide.', $errors->eq(0)->text());
-        $this->assertSame('Cette valeur ne doit pas être vide.', $errors->eq(1)->text());
-        $this->assertSame('Cette ville et ce code postal ne sont pas liés.', $errors->eq(2)->text());
-        $this->assertSame("Cette valeur n'est pas un identifiant valide de ville française.", $errors->eq(3)->text());
-        $this->assertSame("L'adresse est obligatoire.", $errors->eq(4)->text());
-        $this->assertSame("Cette valeur n'est pas valide.", $errors->eq(5)->text());
+        $this->assertSame(7, $errors->count());
+        $this->assertSame('Cette valeur ne doit pas être vide.', $errors->eq(0)->text(), 'First name should be erroneous.');
```

Comment:
Sorry after rebasing, you should have removed the third argument of `assertSame`, my bad!